Documentation/Getting-Started/quickstart.md

```diff
@@ -38,7 +38,7 @@ A simple Rook cluster can be created with the following kubectl commands and [ex
 ```console
 $ git clone --single-branch --branch master https://github.com/rook/rook.git
 cd rook/deploy/examples
-kubectl create -f crds.yaml -f common.yaml -f operator.yaml
+kubectl create -f crds.yaml -f common.yaml -f operator.yaml # -f psp.yaml
```

Comment:
IMO the TLDR is only for the main scenario and they can see later in the doc about the PSP. Since it's commented out I know it won't hurt though if you really want to keep it here.
```suggestion
kubectl create -f crds.yaml -f common.yaml -f operator.yaml
```